cmd/cluster/install.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2023 Huawei Cloud Computing Technologies Co., Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import (
+	"fmt"
+
+	"github.com/openGemini/gemix/pkg/cluster/manager"
+	"github.com/spf13/cobra"
+)
+
+// installCmd represents the install command
+var installCmd = &cobra.Command{
+	Use:   "install",
```

Comment:
将 install 命令的实现改成如下：
```suggestion
	Use:   "install <cluster-name> <version> <topology.yaml>",
```
这样比较方便得看出这些参数是 required，而且是最重要的和 basic 的。


这个 cluster-name 必须是唯一的， 或许需要保存到 ~/.gemix/ 下面，建议调研下，并且对应的 user/key/password/拓扑文件/版本信息，都要跟这个 cluster-name 绑定，后续的 status/start/stop/uninstall 等操作只需要指定 cluster-name，就应该自定读取上述信息。

Reply:
I have fixed it.